### PR TITLE
pt-br translations

### PR DESCRIPTION
**Pull request by CoderSquirrel**
i've started pt-br translation

### CODE REVIEW

File: translations/dde-file-manager_pt_BR.ts [hunk 1]
```diff
@@ -0,0 +1,1439 @@
...
+    </message>
+    <message>
+        <location filename="../filemanager/views/dbookmarkscene.cpp" line="79"/>
+        <source>Disk</source>
+        <translation>Disco</translation>
+    </message>
+    <message>
+        <location filename="../filemanager/views/dbookmarkscene.cpp" line="80"/>
+        <source>Computers in LAN</source>
+        <translation> Computadores na Rede Local</translation>
```
Comment (jhonmike) [inline translations/dde-file-manager_pt_BR.ts]: space

File: translations/dde-file-manager_pt_BR.ts [hunk 2]
```diff
@@ -0,0 +1,1439 @@
...
+    <message>
+        <location filename="../filemanager/dialogs/dialogmanager.cpp" line="227"/>
+        <source>&quot;%1&quot; already exists, please use another name.</source>
+        <translation>&quot;%1&quot; já existe, por favor use outro nome.</translation>
+    </message>
+    <message>
+        <location filename="../filemanager/dialogs/dialogmanager.cpp" line="229"/>
+        <location filename="../filemanager/dialogs/dialogmanager.cpp" line="396"/>
+        <source>Confirm</source>
+        <translation>Cofmirmar</translation>
```
Comment (jhonmike) [inline translations/dde-file-manager_pt_BR.ts]: Confirmar

File: translations/dde-file-manager_pt_BR.ts [hunk 3]
```diff
@@ -0,0 +1,1439 @@
...
+    </message>
+    <message>
+        <location filename="../filemanager/dialogs/dialogmanager.cpp" line="246"/>
+        <source>Delete</source>
+        <translation>Excluir</translation>
+    </message>
+    <message>
+        <location filename="../filemanager/dialogs/dialogmanager.cpp" line="252"/>
+        <source>Empty</source>
+        <translation>Vazio</translation>
```
Comment (jhonmike) [inline translations/dde-file-manager_pt_BR.ts]: não tenho certeza, mas acho que aqui faria mais sentido `Limpar`

File: translations/dde-file-manager_pt_BR.ts [hunk 4]
```diff
@@ -0,0 +1,1439 @@
...
+        <source>%1 d %2 h %3 m %4 s</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>MessageWrongDialog</name>
+    <message>
+        <location filename="../dialogs/messagewrongdialog.cpp" line="12"/>
+        <source>Confirm</source>
+        <translation>Cofmirmar</translation>
```
Comment (jhonmike) [inline translations/dde-file-manager_pt_BR.ts]: Confirmar

File: translations/dde-file-manager_pt_BR.ts [hunk 5]
```diff
@@ -0,0 +1,1439 @@
...
+    </message>
+    <message>
+        <location filename="../filemanager/views/filemenumanager.cpp" line="263"/>
+        <source>Empty Trash</source>
+         <translation>Esvaziar Lixeira</translation>
+    </message>
+    <message>
+        <location filename="../filemanager/views/filemenumanager.cpp" line="264"/>
+        <source>Display as</source>
+        <translation>Exibit como</translation>
```
Comment (jhonmike) [inline translations/dde-file-manager_pt_BR.ts]: Exibir como

File: translations/dde-file-manager_pt_BR.ts [hunk 6]
```diff
@@ -0,0 +1,1439 @@
...
+    <message>
+        <location filename="../filemanager/models/searchfileinfo.cpp" line="295"/>
+        <source>Path</source>
+        <comment>SearchFileInfo</comment>
+        <translation>Caminho</translation>
+    </message>
+    <message>
+        <location filename="../filemanager/models/searchfileinfo.cpp" line="409"/>
+        <source>Searching...</source>
+        <translation>Procurand...</translation>
```
Comment (jhonmike) [inline translations/dde-file-manager_pt_BR.ts]: Procurando...

**Comment by Iceyer** [comment]
We do not accpet Github PR, Please commit code by our Gerrit system:

https://cr.deepin.io/#/admin/projects/dde/dde-file-manager

**Comment by CoderSquirrel** [comment]
@Iceyer i discover that after push here, i joined transifex already and now i'm there

**Pull request closed without merging**